test/v2/correctness/correctness_test.go

```diff
@@ -37,25 +37,27 @@ func getEnvironmentName(environment string) string {
 	return environmentName
 }
 
+// checkAndSetCertVerifierAddress checks whether the input address string is empty, and skips the test if it is
+//
+// If the input address is not empty, this method configures the test client to use the input address as the cert
+// verifier address.
+func checkAndSetCertVerifierAddress(t *testing.T, c *client.TestClient, certVerifierAddress string) {
+	if certVerifierAddress == "" {
+		t.Skip("Requested cert verifier address is not configured")
```

Comment:
first time I see this kind of pattern of skipping a test inside some function. Is this something you've used before? Curious if there would be another way to skip the test earlier and inside the test itself.

Reply:
we talked offline. Would prefer in the future that we don't have this kind of dynamic skipping of tests. But let's move on for now, more tests is always better than no tests!